src/applications/post-911-gib-status/utils/helpers.jsx

```diff
@@ -232,25 +233,9 @@ export function backendErrorMessage() {
 }
 
 export const wizardConfig = [
```

Comment:
We might want to pull this out into its own file because it's so long. Keeping it in here for now is helpful for the diff, though. :man_shrugging:

Reply:
good idea. It was useful while working through things but moving makes sense now. I'm all for reducing file length 🙂